nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/test/resources/conf/new-flow.json

```diff
@@ -0,0 +1 @@
+{"encodingVersion":{"majorVersion":2,"minorVersion":0},"maxTimerDrivenThreadCount":10,"registries":[],"parameterContexts":[],"controllerServices":[],"reportingTasks":[],"templates":[],"rootGroup":{"identifier":"2a2b649d-8538-3239-9965-536b5b993cc5","instanceIdentifier":"13c477b8-0182-1000-31df-454d42e70446","name":"NiFi Flow","comments":"","position":{"x":0.0,"y":0.0},"processGroups":[],"remoteProcessGroups":[],"processors":[],"inputPorts":[],"outputPorts":[],"connections":[],"labels":[],"funnels":[],"controllerServices":[],"variables":{},"defaultFlowFileExpiration":"0 sec","defaultBackPressureObjectThreshold":10000,"defaultBackPressureDataSizeThreshold":"1 GB","flowFileOutboundPolicy":"STREAM_WHEN_AVAILABLE","flowFileConcurrency":"UNBOUNDED","componentType":"PROCESS_GROUP"}}
```

Comment:
It would be helpful to pretty-print these JSON flow definitions to make them easier to read for testing. Alternatively, is it possible to construct the flow definitions in test methods instead of external files?

Reply:
Making the changes to have them set programmatically